config.json

```diff
@@ -47,5 +47,15 @@
     "location": "Pune India",
     "unit": "cel"
   },
-  "settingsIcon": false
+  "settingsIcon": false,
+  "style": {
+    "backgroundColor": "#212121",
+    "messageColor": "#fff",
+    "dateColor": "#d9d9d9",
+    "searchColor": "#fff",
+    "searchBackgroundColor": "#2e2e2e",
+    "squareColor": "#9e9e9e",
+    "squareBackgroundColor": "#2e2e2e"
+  },
+  "backgroundColor": "#212121"
```

Comment:
@benjih Is this `backgroundColor` value even considered? I think this is a mistake right?